docs/docs/winning-over-content-creators.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Winning Over Content Creators"
+issue: https://github.com/gatsbyjs/gatsby/issues/18281
+---
+
+Content creators are probably the heaviest end-users of your Gatsby projects because they create and manage the content used in your project. If you're a developer, you may find that simply talking to content creators about why _you_ like Gatsby is enough to get them interested. Below are topics a content creator cares about and how Gatsby can improve these.
+
+Some specific things that content creators care about include:
+
+- **Using tools they are familiar with.** The content creator probably has tools and workflows that they are already using, like a CMS. Gatsby could be used to integrate and possibly improve that workflow.
```

Comment:
We need to do more to convince the reader that Gatsby can actually improve a CMS workflow. The section below doesn't really cover it, either. Do you have ideas on how to expand on this? e.g. "improve that workflow by integrating with cloud services that automatically deploy content generated in a CMS". This could be a nice tie-in to the Gatsbyjs.com product side, which aims to solve some of these use cases.

```suggestion
- **Using tools they are familiar with.** The content creator probably has tools and workflows that they are already using, like a [CMS](/docs/glossary#cms). Gatsby could be used to integrate and possibly improve that workflow by _____.
```

An alternative suggestion would be to remove the "Gatsby could be used to" sentence and expand on it in the section below.

Reply:
I am currently blank on how to properly convince readers  to improve the workflow, I also feel like it's probably really subjective what 'improvement' is. I do feel like there could be more added though. 
If anyone has any other ideas, please share them 😄!

Reply:
I would suggest a stronger mentioning of the number of plugins that integrate external tools and services into Gatsby projects, and probably, not sure, the work that has been done with the common workflows as an example of how other popular tools can be used with Gatsby. 

Additionally, I think that for this particular guide it may be useful to showcase success stories of creators' projects that use or have been migrated to Gatsby.